packages/valory/skills/generic_scoring_abci/ceramic_db.py

```diff
@@ -69,12 +70,14 @@ def create_user(self, user_data):
 
         fields = self.USER_FIELDS.union(user_data.keys())
 
-        new_user = {
-            field: user_data.get(
-                field, 0 if field in ("points", "current_period_points") else None
-            )
-            for field in fields
-        }
+        new_user = {}
+        for field in fields:
+            if field in ("points", "current_period_points"):
+                new_user[field] = user_data.get(field, 0)
+            elif field in ("tweet_id_to_points"):
+                new_user[field] = user_data.get(field, {})
+            else:
+                new_user[field] = user_data.get(field, None)
```

Comment:
We have code duplication in several skills. There is a ticket to address this, but it has not been prioritized yet.